<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{border:0px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>19</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>54</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,1">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="path_label">
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="path_txt">
          <property name="minimumSize">
           <size>
            <width>739</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Choose File ...</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="browse">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>107</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="proc">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>107</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Process</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>54</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="r1_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Report 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="r2_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>Report 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="r3_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>Report 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="r4_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>Report 4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="r5_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>Report 5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="r6_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>Report 6</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="contents"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
